src/main/java/seedu/address/model/person/UniqueEntityList.java

```diff
@@ -46,6 +47,13 @@ public void add(T toAdd) {
             throw new DuplicatePersonException();
         }
         internalList.add(toAdd);
+
+        //increase id count for customer and driver
+        if (toAdd instanceof Customer) {
```

Comment:
I feel it's kinda weird to increment a global variable at some hidden corner. Ditto, I can help.

Reply:
where do u suggest to put these

Reply:
inside the id manager

Reply:
ok gonna add a method to take in toAdd and transfer the code there

Reply:
removed, move to modelmanager